src/components/global/Playground/index.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+
+import './playground.css';
+
+enum Mode {
+  iOS = 'ios',
+  MD = 'md',
+}
+
+export default function Playground() {
+  const [mode, setMode] = useState(Mode.iOS);
+
+  // TODO FW-741: Load code snippets remotely
+
+  return (
+    <div className="playground">
+      <div className="playground-container">
+        <div className="playground__control-toolbar">
+          {/* TODO FW:-742: Code language switcher */}
+          <div className="playground__control-group">
+            <button
+              type="button"
+              className={
+                'playground__control-button ' + (mode === Mode.iOS ? 'playground__control-button--selected' : '')
```

Comment:
Can we move this logic to above the return function? Same thing with line 33. Similar to how we do https://github.com/ionic-team/ionic-framework/blob/main/core/src/components/datetime/datetime.tsx#L1590-L1592